content/departments/engineering/teams/cody/llm-proxy/index.md

```diff
@@ -9,15 +9,50 @@ In general, we have two LLM Proxy instances running:
 - `completions.sourcegraph.com` - for production usage
 - `completions.sgdev.org` - for development and testing
 
+Contents:
+
+- [Architecture](#architecture)
+- [Service images](#service-images)
+- [Access](#access)
+- [Operation](#operation)
+
 ## Architecture
 
 See [LLM Proxy: working design](https://docs.google.com/document/d/1fAKuYM02vRfn-QAmcu38QWmtQ797g1TA3L6CNs0rFps/edit#) for the current state of LLM proxy architecture.
 
-## Images
+## Service images
 
 Source code for LLM Proxy is in [`sourcegraph/sourcegraph/enterprise/cmd/llm-proxy`](https://github.com/sourcegraph/sourcegraph/tree/main/enterprise/cmd/llm-proxy).
 The image gets built the same as any other Sourcegraph service, i.e. with `insiders` and the standard `main`-branch tags.
 
+## Access
+
+Access to `completions.sourcegraph.com` can be provisioned with the following steps:
+
+1. Go to [site admin: product subscriptions](https://sourcegraph.com/site-admin/dotcom/product/subscriptions)
+2. Find and open a subscription of interest
+3. Under "Cody services":
+   1. Enable access to hosted completions
+   2. If desired, configure a custom rate limit
+   3. Copy the generated access token
+
+On the Sourcegraph instance, configure `llmproxy` as the completions provider:
+
+```json
+{
+  "completions": {
+    "accessToken": "REDACTED",
+    "enabled": true,
+    "model": "claude-v1",
+    "provider": "llmproxy"
+  }
+}
+```
+
+> NOTE: Changes in product subscription, such as enabling access and configuring custom rate limits, may take some time to propagate.
```

Comment:
nit: instead of saying "some", are we comfortable saying an upper limit? e.g. "up to 5 minutes". So it's clear when is the time to reach out for help.

Reply:
The current default interval is 2m:

```suggestion
> NOTE: Changes in product subscription, such as enabling access and configuring custom rate limits, may take around 2 minutes to propagate.
```